### PR TITLE
Update gelfd dependecy to allow uncompressed input

### PR DESCRIPTION
**Pull request by ptqa**
Hi. 

Since gelfd maintainer is not releasing new version of gem for about 2 years (https://github.com/lusis/gelfd/issues/4) I've made a fork and published it to rubygems as gelfd2 (https://github.com/ptqa/gelfd2). I will try to maintain it as long as I can, although there is not a lot of activity.

I've tested this gem on my setup and it seems to work.

### CODE REVIEW

**Comment by bobzoller** [comment]
What's the official word here?  I've now got my own fork of logstash-input-gelf for this reason, and I can see others doing the same.

**Comment by jordansissel** [comment]
I'm open to merging this, but: 

* it currently has a merge conflict
* it needs to be reviewed

File: lib/logstash/inputs/gelf.rb [hunk 1]
```diff
@@ -11,16 +11,8 @@
 # making it a good choice if you already use Graylog2 today.
 #
 # The main use case for this input is to leverage existing GELF
-# logging libraries such as the GELF log4j appender. A library used
-# by this plugin has a bug which prevents it parsing uncompressed data.
```
Comment (jordansissel) [inline lib/logstash/inputs/gelf.rb]: Curious. Why remove this?

Also, if you hope to update the documentation, the docs for this plug now live in `docs/index.asciidoc`

Comment (lambfrier) [inline lib/logstash/inputs/gelf.rb]: Because the bug is fixed in the newer code.

**Comment by rikwasmus** [comment]
#11 would be fixed with this.

**Comment by ptqa** [comment]
@jordansissel I've resolved merge conflict and fixed docs. Is it good for merge now?

**Comment by jamtur01** [comment]
It'd be awesome to see this merged - the Gelf input is largely broken without it. :(

**Comment by pkoretic** [comment]
Any news on this one? 
While I'm asking should this plugin be taken by some other maintainer?

**Comment by os11k** [comment]
Any updates? I would be so happy if this can be merged.

**Comment by ptqa** [comment]
@colinsurprenant @yaauie @robbavey Guys, as we approach 3rd anniversary for this PR, I have rebased it on current master & it's passing all CI tests now. I'm not changing CHANGELOG.md to avoid merge conflicts. Maybe we can merge this PR this year?:)

**Comment by jsvd** [comment]
thanks for the patience and the work you've done here, version 3.3.0 has been published with these changes https://rubygems.org/gems/logstash-input-gelf/versions/3.3.0

**Comment by kolyshkin** [comment]
This made its way to logstash 7.4, released Oct 1, 2019 
https://www.elastic.co/guide/en/logstash/7.4/logstash-7-4-0.html

**Pull request merged**